set.mm

```diff
@@ -98664,12 +98664,13 @@ our Axiom of Choice (in the form of ~ ac2 ).  The proof does not depend
     $d x y $.
     $( Define cardinal number addition.  Definition of cardinal sum in
        [Mendelson] p. 258.  See ~ cdaval for its value and a description.
-       (Contributed by NM, 24-Sep-2004.) $)
+       (Contributed by NM, 24-Sep-2004.)  Use ~ df-dju instead.
+       (New usage is discouraged.) $)
     df-cda $a |- +c = ( x e. _V , y e. _V |->
                    ( ( x X. { (/) } ) u. ( y X. { 1o } ) ) ) $.
 
     $( Cardinal number addition is a function.  (Contributed by Mario Carneiro,
-       28-Apr-2015.) $)
+       28-Apr-2015.)  Use ~ djuex instead.  (New usage is discouraged.) $)
```

Comment:
THis is misleading: |_| is not a function, and djuex is not corresponding to cdafn (there is no corresponding theorem). Proof of theorems using cdafn cannot be transformed straightforward, see ~cda1dif.

Reply:
The connection is approximate at best, so let's pick a better wording. How about:

```
    $( Cardinal number addition is a function.  New proofs are encouraged to
       use the ` |_| ` notation instead of ` +c ` but because the former is not
       a function, the replacement will vary.  (Contributed by Mario Carneiro,
       28-Apr-2015.)  (New usage is discouraged.) $)
```

Reply:
1. I like the pattern: `Use ~ xxx instead.  (New usage is discouraged.) $)` at the end of a comment. It is uniform so makes it easier to search, and it doesn't imply that one should simply replace a label with another one or that no further adapting is necessary.
2. I wouldn't take too much time enhancing comments of statements that are going to be deprecated.

Reply:
Heh, definitely agree with point 2 (especially in this case, where there are only 5 usages and at the current rate I'll be the one to change them all).

As for point 1, I'd already changed it before I saw this comment. Not sure I have a hugely strong opinion which I guess gets me back to point 2.

Reply:
For me, it's Ok now.